src/org/parosproxy/paros/extension/option/OptionsCertificatePanel.java

```diff
@@ -991,10 +999,21 @@ private OptionsCertificatePanel getContentPane() {
 	public void initParam(Object obj) {
 		OptionsParam options = (OptionsParam) obj;
 		OptionsParamCertificate certParam = options.getCertificateParam();
-		useClientCertificateCheckBox.setSelected(certParam.isUseClientCert());

```

Comment:
This can be kept (and remove in the if/else), it's being done always.

Reply:
Sorry for overlooking this (and following comment) in the previous review.

Reply:
Not a problem at all